pkg/monitortests/clusterversionoperator/terminationmessagepolicy/monitortest.go

```diff
@@ -79,7 +113,36 @@ func (w *terminationMessagePolicyChecker) CollectData(ctx context.Context, stora
 
 	// existingViolations is the list of violations already present, don't add to it once we start enforcing
 	existingViolations := map[string]sets.String{
-		"namespace": sets.NewString("pods/<name> <containerType>[<containerName>]"),
+		"namespace": sets.NewString("<containerType>[<containerName>]"),
+		"openshift-catalogd": sets.NewString(
+			"pods/catalogd-controller-manager",
+		),
+		"openshift-cluster-api": sets.NewString(
+			"pods/capa-controller-manager",
+			"pods/capi-controller-manager",
+		),
+		"openshift-cluster-csi-drivers": sets.NewString(
+			"containers[openstack-cinder-csi-driver-operator]",
+			"containers[azure-file-csi-driver-operator]",
+			"containers[shared-resource-csi-driver-operator]",
+			"containers[vmware-vsphere-csi-driver-operator]",
+			"containers[vsphere-webhook]",
+			"pods/kubevirt-csi-node",
+			"pods/nutanix-csi-controller",
+			"pods/nutanix-csi-node",
+			"pods/ibm-vpc-block-csi-controller",
+			"pods/ibm-vpc-block-csi-node",
+			"pods/powervs-block-csi-driver-operator",
+		),
+		"openshift-cloud-controller-manager": sets.NewString(
+			"pods/aws-cloud-controller-manager",
+		),
+		"openshift-machine-api": sets.NewString(
+			"containers[machine-ipam-controller]",
```

Comment:
@JoelSpeed

Reply:
Redirect to @rvanderp3 , PTAL at https://github.com/openshift-splat-team/machine-ipam-controller/pull/7 for the fix